website/content/commands/connect/envoy.mdx

```diff
@@ -75,6 +75,9 @@ Usage: `consul connect envoy [options] [-- pass-through options]`
   In cases where either assumption is violated this flag will prevent the
   command attempting to resolve config from the local agent.
 
+- `envoy_hcp_metrics_bind_port` - By setting this port, Envoy will be configured to
+  send metrics to an HCP collector. By default, this is disabled.
```

Comment:
```suggestion
- `envoy_hcp_metrics_bind_port` - Specifies the port number for Envoy to send metrics to. HCP collectors can collect the metrics at this port. The port is not configured by default. 
```
Attempted to reword this so that we are more concrete about how to configure the param. Better? Worse? No different?

Reply:
Thank you so much for the review , and so sorry for the delay. We made some changes to our approach, see [here](https://github.com/hashicorp/consul/pull/16511/commits/3896dfe280c85865eb6476b505d63527d6f0a375#diff-3944e3639e03d55f8f432002a46d697c061b257ae0fc79e62815fcfd4ce4b47fR78). I tried to reword it based on your suggestions - which I liked. 

I'm not 💯 on the new changes. Would love to get your feedback on the new version, and maybe @freddygv has thoughts here.